src/dex-ui/store/swapSlice/swapSlice.ts

```diff
@@ -201,41 +211,38 @@ const createSwapSlice: SwapSlice = (set, get): SwapStore => {
       const { swap, app } = get();
 
       if (tokenToReceive.tokenMeta.pairAccountId !== tokenToTrade.tokenMeta.pairAccountId) {
-        swap.errorMessage = "Swap Tokens not available for selected tokens";
```

Comment:
Zustand updates require the set function. Do not mutate the store values directly.